bot.py

```diff
@@ -2360,7 +2365,7 @@ async def on_message(message):
 @bot.event
 async def on_raw_reaction_add(payload):
     if payload.user_id not in PI_BOT_IDS:
-        reportsChannel = bot.get_channel(739596418762801213)
+        reportsChannel = bot.get_channel(CHANNELID_REPORTS)
```

Comment:
This should find the `#reports` channel from the guild and `CHANNEL_REPORTS` variable instead of the ID. Allows the code to be more flexible.